src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -14,9 +14,11 @@ redirects:
   - /docs/synthetics/synthetic-public-minion-ips
 ---
 
-New Relic uses a group of minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe, and they are in charge of actually running your monitors.
+You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and are in charge of running your monitors.
 
-Because of this, ensure your firewall allows their network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
+Because of this, ensure your firewall allows your minions to make network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```

Comment:
Where can users find these publicly available addresses? If it's in this doc, link over to it.

Reply:
We will cause confusion between public and private minions if we refer to these as your minions. 
```suggestion
Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```